loader_test.go

```diff
@@ -17,7 +17,7 @@ func TestLoadFile(t *testing.T) {
 	_ = CloseTestDb(TestLoadFileDb)
 	conf := GetTestDbConf(TestLoadFileDb)
 	assert.Nil(t, CreateDatabase(conf))
```

Comment:
Don't need to change it but I assume you're checking for a `nil` error return. The testify libraries have a specific `NoError` function.

Also, can be changed later if you want to, I generally recommend use of `testify/require` rather than `testify/assert`. Exactly the same set of functions, you can `s/assert/require/`. The difference is `assert` functions will continue the test even if the assert fails. That can often result in a `nil` pointer issue that causes a panic and big stack trace, making the test output harder to read. The `require` methods will hard fail and the rest of the specific test won't run (other tests will still get run).